nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/kinesis/stream/ConsumeKinesisStream.java

```diff
@@ -169,31 +170,31 @@ public class ConsumeKinesisStream extends AbstractAwsAsyncProcessor<KinesisAsync
     );
 
     static final PropertyDescriptor KINESIS_STREAM_NAME = new PropertyDescriptor.Builder()
-            .name("kinesis-stream-name")
+            .name("Amazon Kinesis Stream Name")
             .displayName("Amazon Kinesis Stream Name")
```

Comment:
The displayName can be removed as a result of these changes.
```suggestion
```

Reply:
True though the Javadoc says:
> Sets a unique id for the property. This field is optional and if not specified the PropertyDescriptor's name will be used as the identifying attribute. However, by supplying an id, the PropertyDescriptor's name can be changed without causing problems. This is beneficial because it allows a User Interface to represent the name differently.
Params: displayName – of the property
Returns: the builder

I assume though that we can always add `.displayName()` again in the future if needed

Reply:
Yes, that's correct, `displayName` could be added back in the future if needed.